tests/test_movielens.py

```diff
@@ -704,7 +704,7 @@ def split(self, X, y=None, groups=None):
             for _ in range(self.n_splits):
                 yield idx, idx
 
-    cv = CV(n_splits=3, random_state=42)
+    cv = CV(n_splits=3, random_state=None)
```

Comment:
Would you mind pointing me at the error this caused?

We shouldn't merge this change - it would make tests non-deterministic.